qutip/steadystate.py

```diff
@@ -255,12 +243,13 @@ def steadystate(A, c_op_list=[], method='direct', solver=None, **kwargs):
     elif solver not in ['scipy', 'mkl']:
         raise Exception('Invalid solver kwarg.')
 
-    if solver == 'scipy':
-        ss_args = _default_steadystate_args()
-    elif solver == 'mkl':
-        ss_args = _mkl_steadystate_args()
-    else:
-        raise Exception('Invalid solver keyword argument.')
+    # if solver == 'scipy':
+    #     ss_args = _default_steadystate_args()
+    # elif solver == 'mkl':
+    #     ss_args = _mkl_steadystate_args()
+    # else:
+    #     raise Exception('Invalid solver keyword argument.')
+    ss_args = _default_steadystate_args()
     ss_args['method'] = method
```

Comment:
The `solver` value need to be set to `mkl` or `scipy` as needed. It is always `scipy` now.

Reply:
I assumed that it would be up to the user to specify if scipy or mkl was needed? I set scipy as default because I thought it was the most stable solver. Could you tell me what specifically I should set the default as, or what changes can be made?

Reply:
Yes but it is in the variable `solver`, not in `kwargs` so it will not be updated in `ss_args`. Also it should be done before the line `ss_args['info']['solver'] = ss_args['solver']`. Or the info will always says `sicpy` even if mkl was used.

Reply:
Aah, I see the issue. Apologies for the oversight.

Reply:
I've fixed it in the latest commit, please check again. And thanks!